tests/Oryx.BuildImage.Tests/Python/PythonPackageTest.cs

```diff
@@ -103,7 +103,7 @@ public PythonPackageTest(ITestOutputHelper output) : base(output)
                 .ToString();
 
             // Act
-            var image = Settings.BuildImageWithRootAccess;
+            var image = Settings.LtsVersionsBuildImageWithRootAccess;
```

Comment:
terrapin use lts-versions image for packaging so tests should refer to lts-versions images